### DISCUSSION TITLE
ChatGPT以外の言語モデル

### DISCUSSION THREAD

**Discussion by n-yoshikawa** [souyakuchan/LLM_DD_Challenge#3] [Show and tell]
創薬に特化した言語モデルを訓練するのも面白いと思うので、ChatGPT以外の言語モデルを紹介していきます。（随時更新）

## LLaMA派生
Meta（旧Facebook）が開発したLLaMAおよびその派モデルが盛んに開発されています。

- オリジナルのLLaMA https://github.com/facebookresearch/llama
- StanfordのチームがLLaMAを対話用に訓練したAlpaca https://github.com/tatsu-lab/stanford_alpaca
- Alpacaを簡単にチューニングできるAlpaca-LoRA https://github.com/tloen/alpaca-lora
- LLaMAを簡単にチューニングできてマルチモーダルにも対応した（現時点では未公開）LLaMA-Adapter https://github.com/ZrrSkywalker/LLaMA-Adapter

### Lit-LLaMA
Apache 2.0 licenseで公開されているLLaMAの別実装 https://github.com/Lightning-AI/lit-llama

## RWKV
GPTで使われるTransformerではなくRNNを用いた高速な言語モデル https://github.com/BlinkDL/RWKV-LM

## GPT4All
https://github.com/nomic-ai/gpt4all

## Cerebras-GPT
https://huggingface.co/cerebras

他にもあったら教えてください。

**Comment by sekijima** [comment]
UCBのVicuna
https://github.com/lm-sys/FastChat
がある程度の性能が出るオープンソースとして出てきましたね。
https://github.com/lm-sys/FastChat

![Screenshot_20230401-221812~2.png](https://user-images.githubusercontent.com/44859883/229296501-caea99c0-9137-41f4-bee2-bd741f921e04.png)
